### PR TITLE
feat: added Hungarian language

### PR DESCRIPTION
**Pull request by FrancescoGottardi**
- added support for Hungarian language

### CODE REVIEW

**Review by beawar — COMMENTED** [review]
This is a feat, not a chore, since it is adding something.

Have you checked date-fns and tinymce locales to see if the key match?

**Comment by FrancescoGottardi** [comment]
> This is a feat, not a chore, since it is adding something.
> 
> Have you checked date-fns and tinymce locales to see if the key match?



> This is a feat, not a chore, since it is adding something.
> 
> Have you checked date-fns and tinymce locales to see if the key match?

Hi,
i added the one related to the tinymce, the dats fns instead was fine;
- https://github.com/date-fns/date-fns/tree/main/src/locale/hu
- https://github.com/mklkj/tinymce-i18n/blob/master/langs/hu_HU.js

**Comment by beawar** [comment]
> i added the one related to the tinymce
> 
> * https://github.com/mklkj/tinymce-i18n/blob/master/langs/hu_HU.js

I'm sorry, but this is not the official language pack for the version we are using (v6). See the documentation here https://www.tiny.cloud/docs/tinymce/6/ui-localization/ (the link is present also in the LocaleDescriptor type)

Also, if you try the UI and set the UI language to Hungarian, you can see the composer is not translated. And that's because the language pack is not available.
You need to also add the language file inside the assets/tinymce/langs folder, which is used by the Composer component to retrieve the translations

File: assets/tinymce/langs/carbonio-head.svg
Comment (beawar) [inline assets/tinymce/langs/carbonio-head.svg:1]: Wrong move

**Review by beawar — APPROVED** [review]
Looks good now

**Review by keshavbhatt — APPROVED** [review]

**Review by CataldoMazzilli — APPROVED** [review]

**Comment by beawar** [comment]
@Gammax92 @MatteoPerdon Remember to add strings in weblate

**Pull request merged**